v3/docs/07-advanced/h-ss58/index.mdx

```diff
@@ -89,14 +89,14 @@ For the shorter account-index formats, the extra byte represents a far greater p
 
 ## Simple/full address types and account/address/network identifiers
 
-[The registry](https://github.com/paritytech/ss58-registry) expresses the status of the account/address/network *identifiers*.
+[The registry](https://github.com/paritytech/ss58-registry) expresses the status of the account/address/network _identifiers_.
 
-*Identifiers* up to value 64 may be expressed in a *simple* format address, in which the least significant byte (LSB) of the identifier value is expressed as the first byte of the encoded address.
+_Identifiers_ up to value 64 may be expressed in a _simple_ format address, in which the least significant byte (LSB) of the identifier value is expressed as the first byte of the encoded address.
```

Comment:
It's usually not a great idea to use italics for emphasis. Introduction of new terms should be **bolded**, code-based terms should be in `monospace`.